src/legacybot/tools/UnsortedTools.java

```diff
@@ -104,4 +104,23 @@ public static void tryShake(RobotController rc) throws GameActionException{
             }
         }
     }
+
+    /**
+     * Checks if we can make a victory point win, and if so, does.
+     * @param rc
+     * @return The number of bullets we need to collect to win
+     * @throws GameActionException
+     */
+    public static float checkVictory(RobotController rc) throws GameActionException {
+        float possible = rc.getTeamBullets() / rc.getVictoryPointCost();
+        float current = rc.getTeamVictoryPoints();
+
+        if(possible >= 1000 - current){
```

Comment:
```suggestion
        if(possible >= VICTORY_POINTS_NEEDED - current){
```

This would make things more flexible